src/com/tony/server/Main.java

```diff
@@ -1,13 +1,6 @@
 package com.tony.server;
 
-import com.tony.server.response.DeleteResponse;
-import com.tony.server.response.FileContentResponse;
-import com.tony.server.response.HeadResponse;
-import com.tony.server.response.PutPostResponse;
-import com.tony.server.response.OptionsResponse;
-import com.tony.server.response.ImageContentResponse;
-import com.tony.server.response.DirectoryResponse;
-import com.tony.server.response.RedirectResponse;
+import com.tony.server.response.*;
```

Comment:
Don't use wildcard imports, it's better to be explicit with what you are bringing in.

If you don't like the longer list of imports at the top: http://vim.wikia.com/wiki/Folding